### ISSUE TITLE
Loading of reference images - nothing happens

### ISSUE THREAD

**Issue by VolkerH** [macronucleus/chromagnon#11]
Hi,

we were trying to use Chromagnon on Mac OS High Sierra.
When trying import 4-channel volumes (no time series) as `.tif` or `.lsm` file using drag and drop nothing happens. 
As the import seems to be using bioformats I would have expected `tif` files to work.
In what format do we need to provide the reference images?

**Comment by macronucleus** [comment]
Hello, thank you for the post.
Actually, tif and lsm files are read by the `tifffile` package which is faster than bioformats.
I am sorry for the error, which is probably due to an error in Chromagnon interpreting your tif format.
You can also try other original microscope file formats (czi, lif, nd2, oib etc.), then it is probably read by bioformats.
Alternatively deltavision file format (.dv) is read by yet another package which is also faster than bioformats.

If you are wiling to help me to find out the problem in reading your tif or lsm file formats, then here is the procedure to do how.
1. From `Finder`, go to your Chromagnon.app and right click it, and choose "show package contents".
2. Go to "Contents/MacOS", you will find “Chromagnon064Mac”.
3. Now, find the “terminal” application from your mac, and open it.
4. Drag and drop the “Chromagnon” program onto the terminal window, and hit enter. Chromagnon graphical interface will open up.
5. Load your tif or lsm files onto Chromagnon, and hit “Run all”.
6. There will be an error message on the terminal window. Copy the sentence after “Traceback…” and paste it in this thread.

Thank you.

**Comment by GenevieveBuckley** [comment]
When trying to upload a `.tif` image as a reference file:
```
$ /Applications/Chromagnon064Mac.app/Contents/MacOS/Chromagnon064Mac
Traceback (most recent call last):
  File "/Volumes/optim1/matsuda/src/Chromagnon/Chromagnon/Chromagnon/chromagnon.py", line 110, in <lambda>
  File "/Volumes/optim1/matsuda/src/Chromagnon/Chromagnon/Chromagnon/chromagnon.py", line 310, in OnChooseImgFiles
  File "PriCommon/listbox.py", line 250, in addFiles
  File "PriCommon/listbox.py", line 273, in addFile
  File "PriCommon/listbox.py", line 266, in addFile
  File "/Volumes/optim1/matsuda/src/Chromagnon/Chromagnon/Chromagnon/chromagnon.py", line 229, in _load_func
  File "imgio/__init__.py", line 35, in Reader
  File "imgio/__init__.py", line 97, in _switch
  File "imgio/multitifIO.py", line 30, in __init__
  File "imgio/generalIO.py", line 34, in __init__
  File "imgio/multitifIO.py", line 39, in openFile
  File "imgio/multitifIO.py", line 110, in readHeader
  File "imgio/multitifIO.py", line 217, in readChannelInfo
AttributeError: 'list' object has no attribute 'get'
```

When trying to upload a `.lsm` image as a reference file:
```
$ /Applications/Chromagnon064Mac.app/Contents/MacOS/Chromagnon064Mac
/var/folders/gz/6f0v_zhs09x38mb4td_h6py80000gn/T/_MEIZULPsy/imgio/multitifIO.py:237: RuntimeWarning: invalid value encountered in true_divide
Traceback (most recent call last):
  File "/Volumes/optim1/matsuda/src/Chromagnon/Chromagnon/Chromagnon/chromagnon.py", line 110, in <lambda>
  File "/Volumes/optim1/matsuda/src/Chromagnon/Chromagnon/Chromagnon/chromagnon.py", line 310, in OnChooseImgFiles
  File "PriCommon/listbox.py", line 250, in addFiles
  File "PriCommon/listbox.py", line 273, in addFile
  File "PriCommon/listbox.py", line 266, in addFile
  File "/Volumes/optim1/matsuda/src/Chromagnon/Chromagnon/Chromagnon/chromagnon.py", line 231, in _load_func
  File "/Volumes/optim1/matsuda/src/Chromagnon/Chromagnon/Chromagnon/chromagnon.py", line 229, in _load_func
  File "imgio/__init__.py", line 35, in Reader
  File "imgio/__init__.py", line 97, in _switch
  File "imgio/multitifIO.py", line 30, in __init__
  File "imgio/generalIO.py", line 34, in __init__
  File "imgio/multitifIO.py", line 39, in openFile
  File "imgio/multitifIO.py", line 110, in readHeader
  File "imgio/multitifIO.py", line 237, in readChannelInfo
ValueError: cannot convert float NaN to integer
```

**Comment by GenevieveBuckley** [comment]
It's also worth mentioning that having the regex set to look for *only* deltavision files by default was also a little confusing. By the time we'd navigated to the correct folder you couldn't see the `/*.dv*` suffix anymore and it wasn't clear why the directory looked "empty". It'd probably be more user friendly if it accepted by default all readable file formats, or removed the regex altogether and display an error if a user selects a non-supported file type.

**Comment by macronucleus** [comment]
Thank you for the error message and the helpful feedback.
I totally agree that the default suffix `*.dv*`should be changed to just `*` or readable file formats.
Let me look into the cause of the error.

**Comment by macronucleus** [comment]
I think I have located the problem in my code, and therefore put the bug-fixed version in the [release page](https://github.com/macronucleus/Chromagnon/releases/tag/v0.64)
Would you kindly test this version?
Please download `ChromagnonV65Macdebug.dmg` which, upon double clicking, automatically opens terminal window and the graphical interface.
You will see the error message on the terminal window if an error occurred.

It would be great if you let me know if it worked, or showed another error message.

**Comment by GenevieveBuckley** [comment]
I think that's fixed the issue, I don't see an error reported in the terminal now.

**Comment by macronucleus** [comment]
Many thanks for testing it.
These issues should have been solved in v0.65.
I have updated the source code and all binary releases.
Thank you again.
